data/json/items/book/cooking.json

```diff
@@ -171,34 +173,34 @@
     "skill": "cooking",
     "max_level": 1,
     "intelligence": 5,
-    "time": "24 m",
+    "time": "23 m",
     "fun": 1
   },
   {
     "id": "modern_tanner",
     "type": "BOOK",
     "name": "The Modern Tanner",
     "name_plural": "copies of The Modern Tanner",
-    "description": "A in-depth and easy to read guide that details a very modern take on the ancient art of leather tanning.",
+    "description": "A in-depth and easy to read guide with a contemporary perspective on the ancient art of leather tanning.",
     "weight": "568 g",
     "volume": "750 ml",
     "price": 2000,
     "price_postapoc": 5000,
     "material": [ "paper", "leather" ],
     "symbol": "?",
-    "color": "brown",
+    "color": "green",
     "skill": "cooking",
     "required_level": 3,
-    "max_level": 4,
+    "max_level": 5,
     "intelligence": 4,
-    "time": "18 m"
+    "time": "28 m"
   },
   {
     "id": "recipe_alpha",
     "type": "BOOK",
     "name": "PE050 \"Alpha\": Preliminary Report",
-    "name_plural": "PE050 \"Alpha\": Preliminary Report",
-    "description": "This sheaf of papers-dated two weeks before all this started-describes some new chemical formula, and its effects on human subjects.  It's stamped \"APPROVED\"…",
+    "name_plural": "PE050 \"Alpha\": Preliminary Reports",
+    "description": "This sheaf of papers is dated two weeks before all this started.  It describes some new chemical formula, and its effects on human subjects.  It's stamped \"APPROVED\"…",
```

Comment:
There was nothing wrong with the original use of a dash, I'm not sure why the change.